apis/types/capability.go

```diff
@@ -80,6 +80,8 @@ const (
 	OpenapiV3JSONSchema string = "openapi-v3-json-schema"
 	// UISchema is the key to store ui custom schema
 	UISchema string = "ui-schema"
+	// VelaQLTemplate is the key to store velaql view
+	VelaQLTemplate string = "template"
```

Comment:
```suggestion
	VelaQLConfigmapKey string = "template"
```

Reply:
I didn't see you refactor the code who should load the view from K8s and execute it. We should align with that code.

Reply:
Sorry, I don't quite understand what you mean. Can you explain it in detail.